snuba/web/rpc/storage_routing/routing_strategies/storage_routing.py

```diff
@@ -133,7 +133,7 @@ def _construct_hacky_querylog_payload(
                 "end_timestamp": in_message_meta.end_timestamp.seconds,
                 "stats": _get_stats_dict(routing_decision),
                 "status": "0",
-                "trace_id": cur_span.trace_id if cur_span else "no_current_span",
```

Comment:
I think the error in prod was because `cur_span` started to be `None` or empty and it defaulted to `no_current_span`, which is not a UUID and was rejected by the consumer.